example/spec/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -160,6 +161,9 @@ xdescribe('The Cloud Notification Mechanism Kinesis workflow\n', () => {
 
   beforeAll(async () => {
     // populate collections, providers and test data
+    console.log(testConfig.bucket);
```

Comment:
If this just for debugging or are you planning on keeping this?

Reply:
Interesting, I wonder if this is from a bad merge or something. I didn't mean to change this test, so I've reverted back to the version on master.